### PR TITLE
Remove ballerina config and logging modules

### PR DESCRIPTION
**Pull request by warunalakshitha**
## Purpose
Remove old impl of config and unused logging modules

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/jballerina-integration-test/src/test/resources/configurables/multipleTomlModuleSectionsProject/test_module/Dependencies.toml [hunk 1]
```diff
@@ -8,3 +8,5 @@ repository = "local"
 org = "ballerina"
 name = "test"
 version = "0.8.0"
+
```
Comment (HindujaB) [inline tests/jballerina-integration-test/src/test/resources/configurables/multipleTomlModuleSectionsProject/test_module/Dependencies.toml:11]: Shall we remove these new line changes?

File: tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/async_test.bal [hunk 2]
```diff
@@ -11,6 +12,7 @@ public function countSlowly() returns int = @java:Method {
 
 public function main() {
     completeFutureMoreThanOnce();
+    runtime:sleep(1);
```
Comment (HindujaB) [inline tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/async_test.bal:15]: Why do we need a `sleep()` call here?

**Review by HindujaB — COMMENTED** [review]
LGTM

File: tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/async_test.bal (hunk 2 above)
Comment (warunalakshitha) [reply tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/async_test.bal:15]: The test was intermittently failed. In this test we complete the future using multiple threads.
If thread 1 complete, thread 2 complete then the main will return with error. But sometimes main will return before thread 2 complete before error message in system out, so test will finish with an error given "error message not found". Added sleep to make sure main will not exit.

**Review by HindujaB — APPROVED** [review]

**Pull request merged**